huffman/src/bit_span.hpp

```diff
@@ -43,13 +42,10 @@ class bit_span
 
     constexpr auto operator*() const -> reference
     {
-      // TODO: switch to std::bitset once GCC's
-      // std::bitset<_Nb>::reference::operator bool() is constexpr
-      const std::uint8_t byte{
-          // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
```

Comment:
You still need the NOLINT (unless there's some safer way to do this?)

Reply:
Yep I'll take a look either later tonight or tomorrow morning